SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -402,13 +394,27 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 	}
 
 	// If we're the client, we can now ask the server to spawn our controller.
```

Comment:
Can you move this comment closer to what it's referring to?

Reply:
✓